apps/projects/contracts/Projects.sol

```diff
@@ -15,7 +15,7 @@ import "@aragon/os/contracts/lib/math/SafeMath.sol";
   */
 interface Bounties {
     /**
-     * @notice `_sender` will submit a fulfillment for bounty #`_bountyId` on Standard Bounties with the following info: `_data`
+     * @notice `_sender` submit a fulfillment for issue #`_bountyId` with the following info: `_data`
```

Comment:
> This transaction will eventually perform
> 
>   0xb4124cEB3451635DAcedd11767f004d8a28c6eE7 submit a fulfillment for issue #NNNN with the following info: ...
> 
> on 0xblahblahblah

The "_person_ submit a fulfillment" seems like a grammar issue.

Reply:
Woops. Should be 
> `_sender` submitted a fulfillment for issue #`_bountyId` with the following info: `_data`

Reply:
Maybe should be:

> Submitted a fulfillment for issue #`_bountyId` with the following info: `_data`